types/ioredis/index.d.ts

```diff
@@ -700,45 +861,69 @@ declare namespace IORedis {
         hscanStream(key: KeyType, options?: ScanStreamOption): Readable;
         zscanStream(key: KeyType, options?: ScanStreamOption): Readable;
 
-        xack(key: KeyType, group: string, ...ids: string[]): Promise<number>;
+        // xack(key: KeyType, group: string, ...ids: string[]): Promise<number>;
```

Comment:
What's with the commented out bits?

Reply:
These stream commands had nice typings (`xadd` especially) which I kind of have to sacrifice in order to support callbacks. I left the original typings behind commented out in case typescript ever allows us to do`...args` in the middle of an arg list.

Instead of just leaving them like this I'll add comment block above explaining this. Unless you think I should just remove them, I'd be OK with that as well.

Reply:
@SimonSchick any opinion on this or should it be ok to merge?

Reply:
Kinda torn, the history is preserved in git, why keep it as comments/noise?

Reply:
I agree with @SimonSchick here. I don't like the idea of leaving comments if it can be viewed on git.

Reply:
No problem, I'll remove em